README.md

```diff
@@ -159,15 +159,28 @@ Important Notes:
 
 ## Error States
 
+### Error 400: Bad Request, no primary key
+If a WAL record for a table that does not have a primary key is passed through `realtime.apply_rls`, an error is returned
+
+Ex:
+```sql
+(
+    null,                            -- wal
+    null,                            -- is_rls_enabled
```

Comment:
@olirice i think we should update the payload so that `is_rls_enabled` is always a bool instead of null. Realtime server strips this from broadcasted message but it's important when determining whether message should be broadcast to all or `subscriber_ids` array.

so it's never seen by any clients but `is_rls_enabled` is important to determine if all clients should receive this or just authorized users.

Reply:
sounds good. resolved in 177748c0745a6af476c96ce9118986976fe40230

Reply:
@olirice don't forget to update the README here so that `is_rls_enabled` is either true or false.